backend/routes/routes.js

```diff
@@ -47,25 +48,26 @@ router.post("/registration", async (req, res) => {
     if (existingUser) {
```

Comment:
Good one, I would not have thought of this.

Reply:
ended up removing it when i saw that we only used email for logging in. but oh well, now we no how to use or operator in Mongodb :)